File: app/src/main/java/com/sanketvagadiya/Model/ProjectModel.java
package com.sanketvagadiya.Model;


public class ProjectModel {
    String projectName;
    String imageUrl;
    String projectUrl;

    public ProjectModel(String projectName, String imageUrl, String projectUrl) {
        this.projectName = projectName;
        this.imageUrl = imageUrl;
        this.projectUrl = projectUrl;
    }

    public String getProjectUrl() {
        return projectUrl;
    }

    public String getProjectName() {
        return projectName;
    }

    public String getImageUrl() {
        return imageUrl;
    }

}

File: app/src/main/java/com/sanketvagadiya/Model/ProjectModel.kt
package com.sanketvagadiya.Model


class ProjectModel(projectName: String, imageUrl: String, projectUrl: String) {
    var projectName: String
        internal set
    var imageUrl: String
        internal set
    var projectUrl: String
        internal set

    init {
        this.projectName = projectName
        this.imageUrl = imageUrl
        this.projectUrl = projectUrl
    }

}
